src/odemis/gui/comp/overlay/base.py

```diff
@@ -620,7 +620,7 @@ def stop_selection(self):
 
         logging.debug("Stopping selection")
 
-        if max(self.get_height(), self.get_width()) < gui.SELECTION_MINIMUM:
+        if max(int(self.get_height() or 0), int(self.get_width() or 0)) < gui.SELECTION_MINIMUM:
```

Comment:
I don't think you need the `int()`, the following should also work. And can you add a comment explaining that you need the `or zero` in the height or width is None

```suggestion
        if max(self.get_height() or 0, self.get_width() or 0) < gui.SELECTION_MINIMUM:
```